x-pack/solutions/observability/plugins/apm/ui_tests/parallel_tests/service_inventory/service_inventory.spec.ts

```diff
@@ -4,34 +4,34 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { expect } from '@kbn/scout';
-import { test } from '../../fixtures';
-
-const start = '2021-10-10T00:00:00.000Z';
-const end = '2021-10-10T00:15:00.000Z';
+import { expect } from '@kbn/scout-oblt';
+import { test, testData } from '../../fixtures';
 
 test.describe('Service Inventory', { tag: ['@ess', '@svlOblt'] }, () => {
-  test.beforeEach(async ({ browserAuth, page, pageObjects: { serviceInventoryPage } }) => {
+  test.beforeEach(async ({ browserAuth, pageObjects: { serviceInventoryPage } }) => {
     await browserAuth.loginAsViewer();
-    await serviceInventoryPage.gotoDetailedServiceInventoryWithDateSelected(start, end);
-    await serviceInventoryPage.waitForPageToLoad();
+    await serviceInventoryPage.gotoDetailedServiceInventoryWithDateSelected(
+      testData.OPBEANS_START_DATE,
+      testData.OPBEANS_END_DATE
+    );
   });
 
-  test('shows the service inventory', async ({ page, pageObjects: { serviceInventoryPage } }) => {
-    await serviceInventoryPage.gotoDetailedServiceInventoryWithDateSelected(start, end);
-    expect(page.url()).toContain('/app/apm/services');
-    await expect(page.getByRole('heading', { name: 'Services', level: 1 })).toBeVisible();
-  });
+  test('renders page with selected date range', async ({ page }) => {
+    await test.step('shows correct heading', async () => {
+      expect(page.url()).toContain('/app/apm/services');
+      await expect(page.getByRole('heading', { name: 'Services', level: 1 })).toBeVisible();
+    });
```

Comment:
wrapping in `test.step` to avoid openning new browser context 4 times and just split checks unto steps for a better reporting.